staging/src/k8s.io/code-generator/pkg/util/build.go

```diff
@@ -63,7 +65,7 @@ func BoilerplatePath() string {
 	// set up paths to check
 	paths := []string{
 		// works when run from root of $GOPATH containing k8s.io/code-generator
-		filepath.Join(reflect.TypeOf(empty{}).PkgPath(), "/../../hack/boilerplate.go.txt"),
+		filepath.Join(args.DefaultSourceTree(), reflect.TypeOf(empty{}).PkgPath(), "/../../hack/boilerplate.go.txt"),
```

Comment:
This would change this search path to always refer to a copy under GOPATH (if GOPATH is defined), I am not sure if that is the original intention.

I'm wondering if instead it would be better to have BOTH search paths. So that if someone runs in a directory with `k8s.io/code-generator`, the current directory takes precedence over any code-generator package installed under $GOPATH.

```suggestion
		filepath.Join(reflect.TypeOf(empty{}).PkgPath(), "/../../hack/boilerplate.go.txt"),
		filepath.Join(args.DefaultSourceTree(), reflect.TypeOf(empty{}).PkgPath(), "/../../hack/boilerplate.go.txt"),
```